src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts

```diff
@@ -2518,6 +2518,26 @@ export class NotebookEditorWidget extends Disposable implements INotebookEditorD
 			return;
 		}
 
+		// TODO: @Yoyokrazy -- selective webview warmup to improve perf (not in accessiblility mode, and not doing full nb search)
```

Comment:
nit, remove and create an issue for it.

Reply:
see: https://github.com/microsoft/vscode/issues/213432